lte/gateway/configs/docker/mme.conf

```diff
@@ -0,0 +1,240 @@
+################################################################################
```

Comment:
this and the redis.conf files are template file son the lte AGW, are you just hardcoding them for now before adding dynamic templating later?

Reply:
@xjtian replaced mme.conf with conf generation layer in the Docker file.